src/lib.rs

```diff
@@ -239,7 +244,10 @@ impl Client {
         webdriver: &str,
         cap: webdriver::capabilities::Capabilities,
     ) -> Result<Self, error::NewSessionError> {
-        Session::with_capabilities(webdriver, cap).await
+        Session::<hyper_rustls::HttpsConnector<hyper::client::HttpConnector>>::new_rustls(
```

Comment:
I think we'll probably want `Client` to be generic over `C` as well.

Reply:
I think that could create more problems since Element and Form both have Client in them and maybe we could do it like this:

```rust
pub(crate) async fn with_capabilities<T>(
        webdriver: &str,
        mut cap: webdriver::capabilities::Capabilities,
        connector: T,
    ) -> Result<Client, error::NewSessionError>
    where
        T: Clone + hyper::client::connect::Connect + Sync + Send + 'static + Unpin,
```
And maybe indicate with an enum passed to the call to client new to indicate which version we want if both features are enabled and internally use a type parameter in creating the Session
That would also eliminate the need on the user's part to use hyper themselves

Reply:
Actually, maybe the real way to go about this is to store a `Box<dyn Connect + ...>` inside `Session` rather than making it generic. Dynamic dispatch should be fine here, and would make the interface much nicer!